.github/dependabot.yml

```diff
@@ -8,3 +8,11 @@ updates:
       interval: "weekly"
     reviewers:
       - "gardenlinux/garden-linux-maintainers"
+
+  - package-ecosystem: "pip"
+    directory: "/"
```

Comment:
```suggestion
    directory: "/tests"
```

To my understanding this should point to the directory where our pipfile is located

Reply:
Do we need to limit dependabot to check only Pipfile in tests? Why?

Reply:
Checking if dependabot scans recursively or needs explicit paths

Reply:
Yep - you were right

Reply:
fyi, dependabot docs describe this: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#directory